testing/benchmark/.gitignore

```diff
@@ -0,0 +1 @@
+docker_image.auto.tfvars
```

Comment:
suggestion: some of the files in this folder are ignored in the main `.gitignore`. Should we move those lines here for consistency ?

Reply:
Thanks, moved!